packages/core/useDraggable/index.ts

```diff
@@ -135,9 +142,10 @@ export function useDraggable(target: MaybeComputedRef<HTMLElement | SVGElement |
   }
 
   if (isClient) {
-    useEventListener(draggingHandle, 'pointerdown', start, true)
-    useEventListener(draggingElement, 'pointermove', move, true)
-    useEventListener(draggingElement, 'pointerup', end, true)
+    const useCapture = options.useCapture || options.useCapture === null
```

Comment:
This defaults to `false` because `undefined !== null`
Try this instead:
```suggestion
    const useCapture = options.useCapture ?? true
```